### PR TITLE
Improve EteSync: add iOS link, improve description and change links

### PR DESCRIPTION
**Pull request by tasn**
The dav bridge install instructions will soon be moved so better to
use the new links now already (for now, they are just a redirect).

## Description

As the commit message says it improves the EteSync entry by adding a link to the new iOS app, improving the description and changing the soon to be deprecated DAV installation instructions links.

#### Check List 

- [X] I have read and understand [the contributing guidelines](https://github.com/privacytoolsIO/privacytools.io/blob/master/.github/CONTRIBUTING.md).

- [X] The project is [Free Libre](https://en.wikipedia.org/wiki/Free_software) and/or [Open Source](https://en.wikipedia.org/wiki/Open-source_software) Software

* Netlify preview for the mainly edited page: https://deploy-preview-1490--privacytools-io.netlify.com/software/calendar-contacts/

* Code repository of the project (if applicable):
The project is already on privacy-tools, though anyhow, here's the source code: https://github.com/etesync/

### CODE REVIEW

File: _includes/sections/calendar-contacts-sync.html [hunk 1]
```diff
@@ -34,16 +34,20 @@ <h1 id="calendar_contacts" class="anchor"><a href="#calendar_contacts"><i class=
   include cardv2.html
   title="EteSync"
   image="/assets/img/provider/etesync.png"
-  description="<strong>EteSync</strong> is a secure, end-to-end encrypted, and journaled personal information (e.g. contacts and calendar) cloud synchronization and backup for Android and any OS that supports CalDAV/CardDAV. It costs $24 per year to use, or you can host the server yourself for free."
+  description="<strong>EteSync</strong> is a secure, end-to-end encrypted, and privacy-respecting personal information (e.g. contacts and calendar) cloud synchronization and backup for Android, iOS, the web and any other OS (using a CalDAV/CardDAV bridge). It costs $24 per year to use, or you can host the server yourself for free."
```
Comment (Mikaela) [inline _includes/sections/calendar-contacts-sync.html]: How does journaled change to privacy-respecting?

kuva

I think this looks a bit clumsy.

> Android, iOS, the web and any other OS (using a CalDAV/CardDAV bridge).

What are the CalDAV/CardDAV bridges?

I think there is a conflict with the ease of use criteria.

Comment (tasn) [reply _includes/sections/calendar-contacts-sync.html]: Thanks or your feedback.

> How does journaled change to privacy-respecting?
> 

Journaled was the old language we used to use, we stopped using it a while ago. The current language from the website is: "Secure, end-to-end encrypted, and privacy respecting sync for your contacts, calendars and tasks.", so it was just adjusted accordingly.

> kuva
> 
> I think this looks a bit clumsy.

I agree, a second attempt below:

> 
> > Android, iOS, the web and any other OS (using a CalDAV/CardDAV bridge).
> 
> What are the CalDAV/CardDAV bridges?

Fair enough, I agree it's still not clear, but before it looked as if DAV was required for all platforms instead of just the desktop.

> 
> I think there is a conflict with the ease of use criteria.

Could you please elaborate? Not sure I follow.


Second attempt at writing a description:

> <strong>EteSync</strong> is a secure, end-to-end encrypted, and privacy-respecting cloud backup and synchronization software for your personal information (e.g. contacts and calendars). There are native clients for Android, iOS, and the web, and a CalDAV/CardDAV bridge for compatability with DAV clients. It costs $24 per year to use, or you can host the server yourself for free.

Comment (djoate) [reply _includes/sections/calendar-contacts-sync.html]: >  Could you please elaborate? Not sure I follow.

There's [CONTRIBUTING.md](https://github.com/privacytoolsIO/privacytools.io/blob/master/.github/CONTRIBUTING.md) that states this under Software Criteria:

> - Easy to use. Could your mother use that tool or service? Usability is most important.

Comment (djoate) [reply _includes/sections/calendar-contacts-sync.html]: @Mikaela 

> What are the CalDAV/CardDAV bridges?
> 
> I think there is a conflict with the ease of use criteria.

Also, for Nextcloud on the same page, we already mention "CalDAV and contacts sync via CardDAV".

Comment (tasn) [reply _includes/sections/calendar-contacts-sync.html]: Thanks for the clarification @djoate. I misunderstood @Mikaela, and didn't realise the point was about the bridge comment.

As you said, it's also used in the above with NextCloud. I can maybe rephrase that part to "and an adapter layer for most desktop clients.". How would that sound?

Thanks for taking the time to review this.

Comment (tasn) [reply _includes/sections/calendar-contacts-sync.html]: @Mikaela, any thoughts on the above?

Thanks again for taking the time to review this.

Comment (Mikaela) [reply _includes/sections/calendar-contacts-sync.html]: I have lost my train of thought during this time, but I am fine with the second description if you commit it.

I cannot promise when I will be able to look into this for the next time though.

Comment (tasn) [reply _includes/sections/calendar-contacts-sync.html]: Done, thanks for your comment.
